src/duplicates/Duplicate.java

```diff
@@ -30,7 +30,16 @@ public static void removeDuplicates(ArrayList<String> list) {
 				}
 			}
 		}
-		
+	}
+	
+	public static void anotherWayToRemoveDuplicates(ArrayList<String> list) {
+		ArrayList<String> checkedList = new ArrayList<String>();
+		for(String s: list) {
+			if(!checkedList.contains(s)) {
+				checkedList.add(s);
+			}
+		}
+		letterList = checkedList;
```

Comment:
good attempt that is on the very right track. But it can be faster: instead of replacing the whole `list` as `checkedList`, you can write an `else` statement that calls the `ArrayList.remove(int index)` method after the `if `statement in the middle.

Reply:
also, I don't think `letterList` is defined.